steam_wrapper/steam_wrapper.py

```diff
@@ -246,6 +246,26 @@ def configure_shared_library_guard():
         if mode > 1:
             os.environ["LD_BIND_NOW"] = "1"
 
+def setup_proton_extensions():
+    # Create root symlink if it doesn't exist
+    src = STEAM_ROOT + '/.local/share/Steam'
+    dst = STEAM_ROOT + '/.steam/root'
+    proton_dest = src + '/compatibilitytools.d'
+
+    if not os.path.isdir(os.path.dirname(dst)):
+        os.makedirs(os.path.dirname(dst))
+    if not os.path.isdir(os.path.dirname(src)):
+        os.symlink(src, dst)
+
+    # Try to create folder if it doesn't exist
+    os.makedirs(proton_dest, exist_ok=True)
+
+    # Copy extensions if they exist
+    subfolders = [f.path for f in os.scandir("/app/proton") if f.is_dir() ]
+
+    for proton in subfolders:
+        shutil.copy(proton, proton_dest)
+
```

Comment:
In this case actual Proton build wouldn't be updated when flatpak extension is updated.
We need either implement a timestamp check to copy the extension to users directory again if the later is outdated, or just symlink instead of copying.

Reply:
symlinking is not possible due to how proton functions, it needs to be mutable. Although, not sure why it wouldn't be updated when flatpak extensions are updated?